pkg/controllers/deprovisioning/suite_test.go

```diff
@@ -670,6 +670,77 @@ var _ = Describe("Replace Nodes", func() {
 		// and can't delete the node due to the PDB
 		ExpectNodeExists(ctx, env.Client, node1.Name)
 	})
+	It("can replace nodes, do not considers PDB from different namespace", func() {
```

Comment:
```suggestion
	It("can replace nodes, PDB namespace must match", func() {
```

Reply:
Fixed in https://github.com/aws/karpenter-core/pull/89/commits/bb00c2bb2ba6a2ed04ff3a80304b78a1fe4d544d#diff-0065ff411299be24cd997f6ed2a6c1237af42782e705f7847219ef05720af8bf